### PR TITLE
Feature/markov analysis

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
In this pull request, i refactor markov_analysis.py file in practice-2, update function main and add module markov_analysis to main.py
#42

### CODE REVIEW

File: python/practices/practice-3/src/constant/constant_file.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Constant of file markov.txt
+markov_file = 'markov.txt'
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/constant/constant_file.py]: The variable name is not good.

File: python/practices/practice-3/src/path2/markov_analysis.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+def main():
+    """
+    Main function of the program.
+    """
+
+    print(os.path.basename(__file__))
+    file_content = read_file_content(markov_file)
+
+    if file_content is not None:
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path2/markov_analysis.py:80]: Please refactor this condition.

File: python/practices/practice-3/src/path2/markov_analysis.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        number_of_words = int(input('Enter number of words:'))
+        prefix_length = int(input("Enter prefix length:"))
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path2/markov_analysis.py]: Please create new function to do this.

File: python/practices/practice-3/src/constant/constant_file.py (hunk 1 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/constant/constant_file.py]: Done

File: python/practices/practice-3/src/path2/markov_analysis.py (hunk 2 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path2/markov_analysis.py:80]: Done

File: python/practices/practice-3/src/path2/markov_analysis.py (hunk 3 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path2/markov_analysis.py]: Done

**Pull request merged**